client/my-sites/subscribers/components/subscriber-list/subscriber-popover.tsx

```diff
@@ -0,0 +1,54 @@
+import { Gridicon } from '@automattic/components';
+import { Icon, seen, trash } from '@wordpress/icons';
+import classNames from 'classnames';
+import { translate } from 'i18n-calypso';
+import { useCallback, useRef, useState } from 'react';
+import PopoverMenu from 'calypso/components/popover-menu';
+import PopoverMenuItem from 'calypso/components/popover-menu/item';
+
+import './styles.scss';
+
+type SubscriptionPopoverProps = {
```

Comment:
Oh, and I just noticed this type says "Subscription", while the actual component is "**Subscriber**Popover". 😁

Reply:
I'll change this as well.